### DISCUSSION TITLE
How do I disable http access on port 80?

### DISCUSSION THREAD

**Discussion by ghgeiger** [homeassistant-apps/app-cloudflared#238] [Q&A] [answered]
Love the add-on.

I noticed the other day that I could access HA through my cloudflared domain on either port 80 or port 443. I would've guessed that cloudflare would automatically redirect 80 to 443, but it seems it does not. I can't find an obvious setting for this in the zero trust settings, or in the Add-on config. I'm not sure if it matters, but I do not have ssl/tls enabled on my lan HA, just http on port 8123.

**Comment by sfortis** [comment]
Take advance of the cloudflare's WAF and create a rule like this

![image](https://user-images.githubusercontent.com/15342157/198950975-99bb225f-7df3-446a-a1a0-760ba731b6a3.png)

**Comment by brenner-tobias** [comment, marked as answer]
I think the WAF rule for that is a bit of overkill (though it does work). There is a straight forward setting for that in the Cloudflare Dashboard unter "SSL/TLS" and "Edge Certificates" called "Always Use HTTPS", so I recommend enabling that.

<img width="1402" alt="image" src="https://user-images.githubusercontent.com/19689801/199541413-f5d3ec43-181d-4a56-bb83-9c82d545c6be.png">
